app/src/main/java/com/infomaniak/drive/ui/fileList/multiSelect/MultiSelectActionsBottomSheetDialog.kt

```diff
@@ -122,17 +122,21 @@ abstract class MultiSelectActionsBottomSheetDialog(private val matomoCategory: S
     }
 
     protected open fun configureAvailableOffline(): Unit = with(binding) {
```

Comment:
Can't make a suggestion because Github doesn't want, but this is a proposition for this whole function :

```kotlin
    protected open fun configureAvailableOffline(): Unit = with(binding) {

        if (navigationArgs.onlyFolders) {
            availableOffline.isGone = true
            return@with
        }

        availableOffline.apply {
            isVisible = true
            isEnabled = !navigationArgs.onlyFolders
            setOnClickListener { selectOfflineDialogActionCallBack() }
        }

        availableOfflineIcon.isGone = navigationArgs.onlyOffline

        availableOfflineComplete.isVisible = navigationArgs.onlyOffline

        availableOfflineSwitch.apply {
            isChecked = navigationArgs.onlyOffline
            setOnCheckedChangeListener { _, _ -> selectOfflineDialogActionCallBack() }
        }
    }
```

Reply:
I removed the `isEnabled` usage because it's not needed anymore if we return early.